website/sidebars.json

```diff
@@ -7,19 +7,18 @@
       "knowledgebase/getting-started/windows-users",
       "knowledgebase/getting-started/glossary"
     ],
-    "Learn Substrate": [
-      "knowledgebase/learn-substrate/extrinsics",
-      "knowledgebase/learn-substrate/tx-pool",
+    "Substrate Key Concepts": [
+      "knowledgebase/runtime/index",
+      "knowledgebase/learn-substrate/extrinsics",      
       "knowledgebase/learn-substrate/account-abstractions",
+      "knowledgebase/learn-substrate/tx-pool",
       "knowledgebase/learn-substrate/session-keys",
       "knowledgebase/learn-substrate/weight",
       "knowledgebase/learn-substrate/off-chain-features"
     ],
-    "Runtime": [
-      "knowledgebase/runtime/index",
-      "knowledgebase/runtime/primitives",
-      "knowledgebase/runtime/frame",
+    "Runtime Principles": [
```

Comment:
```suggestion
    "Substrate Runtime": [
```

Reply:
`Principles` sounds like conceptual and high level. These topics below are somewhat on actual development/implementation level. So I won't be calling this principles.

But feel free to update to other terms if you don't like the name `Substrate Runtime`.

Reply:
Thanks for the review @jimmychu0807 . Good point on the name. What about Runtime Development ?

Reply:
Sure `Runtime Development` sounds good.